### PR TITLE
IDEX-4034: Refine part tabs

### PR DESCRIPTION
**Pull request by vzhukovskii**
@vparfonov @ashumilova 

tab-items

### CODE REVIEW

File: ide/che-core-ide-api/src/main/resources/org/eclipse/che/ide/api/parts/partstack.css [hunk 1]
```diff
@@ -9,50 +9,42 @@
...
-    fill 0.1s ease-in-out,
-    stroke 0.1s ease-in-out;
+                        fill 0.1s ease-in-out,
+                        stroke 0.1s ease-in-out;
     transition: color 0.1s ease-in-out,
-    fill 0.1s ease-in-out,
-    stroke 0.1s ease-in-out;
+                fill 0.1s ease-in-out,
+                stroke 0.1s ease-in-out;
 }
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-api/src/main/resources/org/eclipse/che/ide/api/parts/partstack.css]: Smth with formatting

File: ide/che-core-ide-api/src/main/resources/org/eclipse/che/ide/api/parts/partstack.css [hunk 2]
```diff
@@ -9,50 +9,42 @@
...
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-api/src/main/resources/org/eclipse/che/ide/api/parts/partstack.css]: Smth with formatting

**Comment by vzhukovskii** [comment]
@vitaliy-guliy fixed

File: ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css [hunk 3]
```diff
@@ -17,25 +17,26 @@
...
-
     background-color: tabsPanelBackground;
 }
 
 .ide-work-bench-tool-panel-left {
-    width: 24px;
-    border-right: 1px solid tabBorderColor;
-    padding-top: 26px;
+    height: 24px;
+    width: 10000px;
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css]: What is it?

File: ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css [hunk 4]
```diff
@@ -17,25 +17,26 @@
...
+    transform-origin: 12px 12px;
+    border-top: 1px solid tabBorderColor;
 }
 
 .ide-work-bench-tool-panel-right {
-    width: 24px;
-    border-left: 1px solid tabBorderColor;
-    padding-top: 31px;
+    height: 24px;
+    width: 10000px;
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css]: We should not do this

File: ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css (hunk 3 above)
Comment (vzhukovskii) [inline ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/workspace/perspectives/general/WorkBench.css]: this is the easiest way to correctly rotate tab panels

**Comment by ashumilova** [comment]
+1 only after 4.0.0-beta release.

**Comment by skabashnyuk** [comment]
ok

**Comment by vzhukovskii** [comment]
There was created new PR #678 to avoid streching git history from this fork.

**Pull request closed without merging**